src/components/forms/Form/Form.stories.tsx

```diff
@@ -174,3 +175,45 @@ export const addressForm = (): React.ReactElement => (
     </Fieldset>
   </Form>
 )
+
+export const signInForm = (): React.ReactElement => (
+  <Form onSubmit={mockSubmit} large>
+    <Fieldset legend="Sign In">
+      <span>
+        or <a href="javascript:void(0);">create an account</a>
+      </span>
+
+      <Label htmlFor="username">Username or email address</Label>
+      <TextInput id="username" name="username" type="text" />
```

Comment:
just to be in sync with the USWDS version, can we add these attributes to this input `autocapitalize="off" autocorrect="off"`